Training/python/2018v1/train.yaml

```diff
@@ -10,7 +10,7 @@ path_to_mlflow: mlruns
 experiment_name: ???
 
 # setup
-scaling_cfg: ../../configs/ShuffleMergeSpectral_trainingSamples-2_files_0_50.json # for DataLoader initialisation
+scaling_cfg: ../../configs/ShuffleMergeSpectral_trainingSamples-2_rerun_files_0_19.json # for DataLoader initialisation
```

Comment:
I suggest using the latest file `ShuffleMergeSpectral_trainingSamples-2_files_0_498.json`. Or it has some known issues?

Reply:
@lguzzi Is ShuffleMergeSpectral_trainingSamples-2_files_0_498.json scaling for the ShuffleMergeSpectral_trainingSamples-2_rerun version of S&M?

Reply:
yes, they were run on the rerun version of the ntuples